homeassistant/components/husqvarna_automower_ble/config_flow.py

```diff
@@ -60,7 +61,28 @@ async def async_step_bluetooth(
         self.address = discovery_info.address
         await self.async_set_unique_id(self.address)
         self._abort_if_unique_id_configured()
-        return await self.async_step_confirm()
+        return await self.async_step_bluetooth_confirm()
+
+    async def async_step_bluetooth_confirm(
```

Comment:
Also, Do we now have 

Discovery -> bluetooth_confirm form -> confirm form -> create_entry?
User form -> confirm form -> create_entry?

Let's try to remove that confirm now, since clicking `submit` on the page to fill in the PIN sounds like a good enough confirm to me. Same for the user flow.